packages/api-explorer/src/lib/oas-to-har.js

```diff
@@ -156,15 +156,30 @@ module.exports = (
 
   const schema = getSchema(pathOperation, oas) || { schema: {} };
 
+  function stringify(json) {
+    // Default to JSON.stringify
+    har.postData.text = JSON.stringify(typeof json.RAW_BODY !== 'undefined' ? json.RAW_BODY : json);
+  }
+
   if (schema.schema && Object.keys(schema.schema).length) {
     // If there is formData, then the type is application/x-www-form-urlencoded
     if (Object.keys(formData.formData).length) {
       har.postData.text = querystring.stringify(formData.formData);
     } else if (isPrimitive(formData.body) || Object.keys(formData.body).length) {
-      // Default to JSON.stringify
-      har.postData.text = JSON.stringify(
-        typeof formData.body.RAW_BODY !== 'undefined' ? formData.body.RAW_BODY : formData.body,
-      );
+      try {
```

Comment:
Can you add some comments about what this is doing? It's not really obvious when it falls into this case